src/Morpho.sol

```diff
@@ -26,7 +35,7 @@ import {SafeTransferLib} from "./libraries/SafeTransferLib.sol";
 /// @author Morpho Labs
 /// @custom:contact security@morpho.org
 /// @notice The Morpho contract.
-contract Morpho is IMorpho {
+contract Morpho is IMorphoStaticTyping {
```

Comment:
it's weird t not have `Morpho is IMorpho`

I would have named IMorpho something like IMorphoStructs and IMorphoStaticTyping IMorpho instead

Reply:
I prefer having it unnatural for readers and natural for integrators rather than the other way around

Reply:
normal clash, as I'm not an integrator and you are (in some sense) 😄